index.html

```diff
@@ -1051,16 +1051,22 @@ <h3>Data delivery</h3>
               and {{PressureRecord/[[Time]]}} set to |timestamp|.
             </li>
             <li>
-              If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
-              set observer.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] to |record|
-              and [=iteration/continue=].
+              Increase |observer|.{{PressureObserver/[[ChangesCountMap]]}}[|source|].
             </li>
             <li>
               If running [=passes rate obfuscation test=] with |observer| and |source| returns false:
               <ol>
+                <li>
+                  If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
```

Comment:
Although this is correct, it looks a bit confusing to me since this step as well as the next one have the same "Set [[AfterPenaltyRecordMap]][source] to |record|" bit.

What if you use an auxiliary variable so that it looks like this:
* Let |replacingRecord| be true if [[AfterPenaltyRecord]][source] exists, and false otherwise.
* Set [[AfterPenaltyRecord]][source] to |record|.
* If |replacingRecord| is true, continue.

Reply:
I felt the same way, but couldn't come up with a solution like yours.

Reply:
|shouldReplaceRecord| would be a better name. Otherwise it doesn't sound like a boolean